subworkflows/local/gtdbtk.nf

```diff
@@ -90,6 +90,7 @@ workflow GTDBTK {
     GTDBTK_CLASSIFYWF (
         ch_filtered_bins.passed.groupTuple(),
         ch_db_for_gtdbtk,
+        params.gtdbtk_pplacer_scratchoff ? false : true
```

Comment:
Missing a comma, looks like this is why the checks failed

Reply:
Ah damn it